### PR TITLE
Prepared AsciiDoc document and ESM build

### PR DESCRIPTION
**Pull request by benjaminleonard**
WIP

### CODE REVIEW

File: scripts/convert-tokens.ts [hunk 1]
```diff
@@ -306,51 +305,6 @@ StyleDictionary.registerFormat({
   },
 })
 
-type ColorGroup = { [colorValue: string]: TransformedToken[] }
```
Comment (benjaminleonard) [inline scripts/convert-tokens.ts:309]: We're not using these right now, I think can reduce some complexity by removing. (Also they don't gel well with regular dedent)

File: styles/src/tokens.json [hunk 2]
```diff
@@ -1532,7 +1532,8 @@
         "colors": "enabled",
         "core": "enabled",
         "purple-accent": "enabled",
-        "main": "enabled"
+        "main": "enabled",
+        "green-accent": "disabled"
```
Comment (benjaminleonard) [inline styles/src/tokens.json:1536]: Might need to resync with the figma tokens plugin. I suspect this change is a mistake.

File: tsup.config.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ export default defineConfig({
   entry: ['components/src/index.ts'],
   outDir: 'components/dist',
   external: ['react/jsx-runtime'],
-  splitting: false,
```
Comment (benjaminleonard) [inline tsup.config.ts:15]: Supported in ESM and on by default

File: scripts/convert-tokens.ts (hunk 1 above)
Comment (david-crespo) [reply scripts/convert-tokens.ts:309]: Love it

**Comment by augustuswm** [comment]
:rocket: PR was released in [`v1.6.0`](https://github.com/oxidecomputer/design-system/releases/tag/v1.6.0) :rocket:

**Pull request merged**